README.md

```diff
@@ -161,3 +161,7 @@ We can build the contract using rustc:
 ```bash
 RUSTFLAGS='-C link-arg=-s' cargo build --target wasm32-unknown-unknown --release
 ```
+
+## License
+This repository is distributed under the terms of both the MIT license and the Apache License (Version 2.0).
+See [LICENSE](LICENSE) and [LICENSE-APACHE](LICENSE-APACHE) for details.
```

Comment:
Should we also rename it to `MIT-LICENSE`?

Reply:
@vgrichina preferred it be called LICENSE only. I think it may be standard to have one file named that way. Beyond that, I do not have strong feelings.